Arduino/SIOT_MCU_serial/SIOT_MCU_serial.ino

```diff
@@ -6,6 +6,10 @@
 #include <siot_serial.pb.h>
 #include <point.pb.h>
 
+#include <PacketSerial.h>
```

Comment:
using <> includes are generally reserved for system headers, I see that its not just this patch where we are using it for application headers. But I wonder if we should consider this convention.

Reply:
Thanks for the review -- this is a good question -- I'm not real familiar with Arduino, but I think the convention is to use < > for library headers, which PacketSerial.h is in this case:

https://github.com/simpleiot/firmware/blob/master/envsetup.sh#L3

I could be wrong -- will need to look at a some examples to know for sure.

This gives some more information: https://stackoverflow.com/questions/21593/what-is-the-difference-between-include-filename-and-include-filename

so PacketSerial is not really not part of the Standard library, but it is a library header file. What is the best practice in that case?

Reply:
here is an example from Arduino docs: https://www.arduino.cc/reference/tr/language/structure/further-syntax/include/

Reply:
ok and how about `#include <siot_serial.pb.h>`

Reply:
That is technically a SIOT library, and not a local file to the project as well:

https://github.com/simpleiot/firmware/tree/master/libraries/pb

The idea is that the SIOT protobuf stuff would be used for a number of different firmware projects.

Maybe we're overusing libraries :-)

Reply:
Arduino code uses libraries extensively. I think the approach makes since in this context.